### PR TITLE
PD-324: add enterprise info to minio app articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR makes changes to the SCALE Tutorial Applications articles navigation. It adds a new expand section for Enterprise Applications, and changes the weights of the other tutorial articles to have them display below the new section. The new _index.md article still needs a nuanced statement about Enterprise Apps verses Official Chart Apps.

It moves the existing EnterpriseApplicationsSCALE.md article to the new section, and adds a new article for configuring MinIO Official Enterprise using the minio wizard.

It updates content in the MinIO Official Charts tutorials. Adds a procedure overview to the MinioManualUpdate.md article, and updates content in MinIOClusterin.md article and replaces or removes out of date docker image screenshots.

It makes minor corrections in the Ui Ref article AppsScreensSCALE.md.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+The MinIO Enterprise application provides limited configuration settings for Enterprise systems. 
+These settings allow Enterprise users to deploy a basic MinIO application in their environments. 
+
+We recommend that TrueNAS SCALE Enterprise (HA) systems not deploy applications.
+{{< / enterprise >}}
+
+
+## First Steps
+
+If your system includes shares (SMB, NFS, iSCSI), disable the share service before adding and configuring a the MinIO application.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:27]:
```suggestion
If your system has active sharing configurations (SMB, NFS, iSCSI), disable these sharing services in **System Settings > Services** before adding and configuring the MinIO application.
```

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+{{< / enterprise >}}
+
+
+## First Steps
+
+If your system includes shares (SMB, NFS, iSCSI), disable the share service before adding and configuring a the MinIO application. 
+After completing the installation and starting MinIO, enable the share service.
+
+If you created a dataset for the MinIO shares and it has the same path as the MinIO application, disable host path validation. 
+To use host path validation create a dataset for the application with a unique path. For example, */pool/shares/minio* for the share and */pool/apps/minio* for the application.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:31]:
```suggestion
To use host path validation, create a dataset for the application with a unique path. For example, */pool/shares/minio* for the share and */pool/apps/minio* for the application.
```

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Configuring MinIO Enterprise
+{{< hint info >}}
+This basic procedure covers the required MinIO Enterprise settings.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:35]:
```suggestion
This basic procedure covers the required Enterprise MinIO App settings.
```

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+If you created a dataset for the MinIO shares and it has the same path as the MinIO application, disable host path validation. 
+To use host path validation create a dataset for the application with a unique path. For example, */pool/shares/minio* for the share and */pool/apps/minio* for the application.
+
+## Configuring MinIO Enterprise
+{{< hint info >}}
+This basic procedure covers the required MinIO Enterprise settings. 
+It does not provide instructions for optional settings.
+{{< /hint >}}
+
+SCALE Enterprise single controller systems with the applications and virtual machines license have access to the **MinIO Official Enterprise** widget. Go to **Apps**, select the pool to use for applications, if not already selected, then click on **Available Applications** screen.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:39]:
```suggestion
SCALE Enterprise single controller systems with the applications and virtual machines license feature have access to the **MinIO Official Enterprise** widget. Go to **Apps**, select the pool to use for applications, if not already selected, then click on the **Available Applications** screen.
```

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+![MinIOEnterpriseWidget](/images/SCALE/22.12/MinIOEnterpriseWidget.png "MinIO Enterprise Widget")
+
+Click **Install** on the **MinIO Official Enterprise** widget to open the **minio** installation wizard.
+
+![MinIOEnterpriseAppNameAndVersion](/images/SCALE/22.12/MinIOEnterpriseAppNameAndVersion.png "MinIO Enterprise Appliation Name and Version")
+
+Enter a name for your MinIO application deployment if not using the default value. **Version** populates with the current MinIO version.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:47]:
```suggestion
Enter a name for your MinIO application deployment. A default value is provided but can be overriden. **Version** populates with the current MinIO version.
```

File: content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md [hunk 6]
```diff
@@ -11,55 +11,83 @@ tags:
...
 
 ## Configuring MinIO
 
-You can configure the MinIO application using either the **Launch Docker Image** option or the **Install** option on the MinIO application widget on the **Available Applications** tab. The best practice is to use the MinIO widget **Install** option, which pre-populates the version and other configuration settings the **Launch Docker Image** option requires you to populate.
+You can configure the MinIO application using either **Launch Docker Image** at the top right of the **Apps** screen, or the **Install** option on the **MinIO Official Charts** application widget. found on the **Available Applications** screen.
 
-On your first node, go to **Apps**, click **Available Applications**, locate the MinIO widget, and click **Install** to open the **minio** wizard.
+We recommend using the **Install** option on the **MinIO** application widget. 
+
+If your system has sharing (SMB, NFS, iSCSI) configured, disable the share service before adding and configuring a new MinIO deployment.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md:49]: update according to the comment/suggestion above
```suggestion
If your system has sharing (SMB, NFS, iSCSI) configured, disable the share service before adding and configuring a new MinIO deployment. 
```

File: content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md [hunk 7]
```diff
@@ -11,55 +11,83 @@ tags:
...
+You can configure the MinIO application using either **Launch Docker Image** at the top right of the **Apps** screen, or the **Install** option on the **MinIO Official Charts** application widget. found on the **Available Applications** screen.
 
-On your first node, go to **Apps**, click **Available Applications**, locate the MinIO widget, and click **Install** to open the **minio** wizard.
+We recommend using the **Install** option on the **MinIO** application widget. 
+
+If your system has sharing (SMB, NFS, iSCSI) configured, disable the share service before adding and configuring a new MinIO deployment. 
+After completing the installation and starting MinIO, enable the share service.
+
+If the dataset for the MinIO share has the same path as the MinIO application, disable host path validation before starting MinIO. 
+To use host path validation set up a new dataset for the application with a completely different path. For example, for the share */pool/shares/minio* and for the application */pool/apps/minio*.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md:53]:
```suggestion
To use host path validation, set up a new dataset for the application with a completely different path. For example, for the share */pool/shares/minio* and for the application */pool/apps/minio*.
```

File: content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md [hunk 8]
```diff
@@ -11,55 +11,83 @@ tags:
...
 ![AppsMinIOAppNameAndScaling](/images/SCALE/22.12/AppsMinIOAppNameAndScaling.png "MinIO Setup Wizard")
 
-First, enter a name in **Application Name** (e.g. *minio* for a standard configuration or *minio-distributed* for a distributed MinIO configuration). 
+Select the update strategy from the **Minio update strategy** dropdown list. The best practice is to keep the default **Create new pods and then kill old ones** setting.
 
-Select the update strategy you want to use from the **Minio update strategy** dropdown list. The best practice is to select **Create new pods and then kill old ones**.
+Next, enter the **MinIO Configuration** settings.
 
-Select **Enable Distributed Mode** if you want to set up a cluster of SCALE systems in a distributed cluster. 
+Select **Enable Distributed Mode** if setting up a cluster of SCALE systems in a distributed cluster.
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/Apps/MinIOClustersSCALE/MinIOClustering.md:73]:
```suggestion
Select **Enable Distributed Mode** when setting up a cluster of SCALE systems in a distributed cluster. 
```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Looking good! I've got a few polish comments/suggestions.

**Comment by micjohnson777** [comment]
Thank you for the polish points! I did have to fix some stray ``` in the first article but other than that, all comments committed!

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**